### PR TITLE
support tx pulling from master

### PR DESCRIPTION
**Pull request by mattkime**

### CODE REVIEW

File: src/commands/txCommands/pull.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const pullResourceContent$ = branch =>
 		// 2. parse local trn content and grab updates
 		.flatMap(([lang_tag, content]) =>
 			Rx.Observable.zip(
-				[lang_tag],content,
+				[lang_tag], [content],
```
Comment (mattkime) [inline src/commands/txCommands/pull.js]: I wish I knew why this was needed at this particular moment. /shrug

**Comment by mmcgahan** [comment]
is there a bug description for this? hard to review without context

**Comment by mattkime** [comment]
Sorry, I thought this was discussed in the group a few times.

On occasion, we need to pull tx content from master. The anticipated use case for this is translation correction that happens in the mup-web-master project. The unanticipated use for this is translated content that only appears in mup-web-master. (Although the most recently time this was requested it appeared that the content did indeed exist in the po files)

**Comment by sadafie** [comment]
Nice! Usage docs should be updated

**Pull request merged**